### DISCUSSION TITLE
WatsonServer in a WindowsServices

### DISCUSSION THREAD

**Discussion by DevOfPixVert** [dotnet/WatsonWebserver#63] [General]
Hello, 

     Is it possible to use WatsonWebServer into a Windows Services and implement multi threading process ?

     For example, use webserver to receive Json request in order to execute async tasks

Best regards
Steph

**Comment by jchristn** [comment]
Hi @StephOfPixVert yes absolutely.  Watson is async by design and each request creates a separate async task.
